src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_file_scenarios.py

```diff
@@ -91,7 +91,7 @@ def test_storage_file_main_scenario(self, resource_group, storage_account):
 
         self.storage_cmd('storage share update --name {} --quota 3', account_info, s1)
         self.storage_cmd('storage share show --name {}', account_info, s1) \
-            .assert_with_checks(JMESPathCheck('properties.quota', 3))
+            .assert_with_checks(JMESPathCheck('quota', 3))
```

Comment:
If the response has been flatten in track2 SDK, can you add transformer to avoid output breaking change?